notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>from point gamma(t) to point gamma(s) </p><p> </p><p>--&gt; t and s are real values (for example 0.2 and 0.8), gamma(t) and gamma(s) belong to the manifold. (e.g. gamma(0.2) and gamma(0.8)).</p><p> </p><p>what that initial point will look like if it is transported along the curve&nbsp;γ</p><p>--&gt; did you mean: what that initial tangent vector will look like if it is transported along the curve gamma from point gamma(t) to point gamma(s)?</p><p> </p><p>T_gamma(t)M and T_gamma(s)M for the tangent spaces.</p><p> </p><p>This is similar to using a differential equation:</p><p>--&gt; actually, this is exactly like solving a differential equation: the differential equation is the one given by nabla _\dot gamma(t) X = 0.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Thanks! for the first comment, what i think you are saying is:</p><ol><li>change "from point t to point s" to "from point gamma(t) to point gamma(s)"</li></ol><p> </p><p>for the last comment: </p><p>what you are saying is that the differntial equation is "nabla _\dot gamma(t) X = 0.". So essentially, we are trying to find a tangent vector that is an element of a parallel vector field along that curve, with that initial point/tangent vector. Here is what i said:</p><p> </p><p>Conceptually, this is exactly like solving a differential equation: we can find a "solution" (in this case: the transported vector) to the equation $\nabla_{\dot \gamma(t)} X = 0$ if we are given initial conditions (in this case: an initial point and tangent vector). In other words, the parallel transport is determining the parallel vector field along the curve $\gamma(t)$ and then determining which element of the vector field lies at the final point $\gamma(s)$</p>

Reply:
1. Yes!

For the lasst comment: yes, while actually the differential equation is : $\nabla_{\dot \gamma(t)} X(\gamma(t)) = 0$ (my bad), to be solved in X.

Reply:
<p>Just to make sure: by adding the $X(\gamma(t)$ are you just indicating that $X$ is parallel along the curve $\gamma(t)$?</p>